ecosystem/sep-0031.md

```diff
@@ -36,6 +36,76 @@ Typically, the Sending and Receiving Clients reside in different regulatory juri
 
 Alice in Nigeria wants to send money to Bob in Europe. Alice signs up with NigeriaPay to make this payment to send money directly into Bob’s bank account. Bob doesn’t need to do anything, or know anything about this payment, besides letting Alice know what his bank account information is. Alice only needs to deal with her anchor (NigeriaPay). Alice passes this information and her money to NigeriaPay, and NigeriaPay sends those funds to the EuroPay Anchor service, and Europay deposits those funds into Bob’s bank account.
 
+## Diagrams
+
+The diagrams provided offer a detailed view of various flows that are possible using this protocol. Note that these diagrams are opinionated. Variations of the these flows may also be sufficient.
+
+### Completing a Transaction with a Firm Quote
+
+This diagram demonstrates the interactions between the entities involved in a successful transaction. Specifically it uses the [Receiving Anchor Asset Conversion](#receiving-anchor-asset-conversion) strategy defined described later in the document.
+
+```mermaid
```

Comment:
This diagram looks great. 👏🏻 

One thing I noticed is that it renders with a massive blank area at the bottom of the diagram. Do you know why this is? Is this a @github bug, or something to do with how we've defined the diagram do you think?

Here's a screenshot of what I see:
<img width="546" alt="Screen Shot 2022-03-08 at 1 12 31 PM" src="https://user-images.githubusercontent.com/351529/157325999-d617ad7a-8c7b-4f8f-beb9-c4bc4ed9d7bd.png">

Reply:
I see the same thing, but I don't see anything wrong with the diagram syntax. Let me google a bit to see if this is a known issue.

Reply:
https://github.com/mermaid-js/mermaid/issues/1984